packages/features/ee/users/components/UserForm.tsx

```diff
@@ -95,7 +95,7 @@ export const UserForm = ({
       },
       timeZone: defaultValues?.timeZone || "",
       weekStart: {
-        value: defaultValues?.weekStart || weekStartOptions[0].value,
+        value: defaultValues?.weekStart || weekStartOptions[1].value,
```

Comment:
Nah, this is not correct. This is not solved the issue. The correct week start value is saved in the database, but there is a problem in the frontend

Reply:
Hey ,Are there any docs for how the code is written or i have to figure out on my own ?  I am new to this repo.